sigle/next.config.js

```diff
@@ -13,6 +13,8 @@ const nextConfig = {
     FATHOM_SITE_ID: process.env.FATHOM_SITE_ID,
     NEXT_PUBLIC_SENTRY_DSN: process.env.NEXT_PUBLIC_SENTRY_DSN,
     NEXT_PUBLIC_POSTHOG_TOKEN: process.env.NEXT_PUBLIC_POSTHOG_TOKEN,
+    API_URL_DEV: process.env.API_URL_DEV,
```

Comment:
Why do we have 2 env variables here? Wdyt about merging both of them to `API_URL`?

Reply:
Why do you want to have a check there? As the config is coming from the .env file (which is different in dev and prod) the value will not be the same.
We can just declare one value, the same way as we do with `APP_URL` :)

Reply:
ahh right yes of course, totally makes sense :)